includes/class-gdpr.php

```diff
@@ -280,6 +280,22 @@ private function define_public_hooks() {
 		add_action( 'wp', array( $requests_public, 'request_confirmed' ) );
 		add_action( 'wp_ajax_gdpr_send_request_email', array( $requests_public, 'send_request_email' ) );
 		add_action( 'wp_ajax_nopriv_gdpr_send_request_email', array( $requests_public, 'send_request_email' ) );
+
+		add_action( 'init',  [ $this, 'remove_gdpr_plugin_hooks' ] );
+	}
+
+	/**
+	 * Removes any GDPR plugin hooks that will be overridden by custom peake code.
```

Comment:
Mind just explaining why we want this in the plugin?

Reply:
My aim was to remove the `client-mu-plugins/gdpr` directory entirely, and that would mean moving all of the code in there into this repo. 

The reason for this is that having two locations is really confusing, so I'd like to remove that as a 'thing' asap.

We haven't yet got a fully working User Delete process, so this is what happens currently - but needs to be redone (probably totally redone) to actually implement that functionality. In the mean time, the existing functionality, such as it is, needs to remain in order to avoid regression.

This is the justification for this admittedly rather weird ticket. 

Can drop this and keep this functionality in the `client-mu-plugins/gdpr` directory though until implemented properly.

Reply:
I think we should aim to keep this plugin agnostic (would be cool to offer to contribute our changes back at some point even if we feel it will fall on deaf ears). What I've been doing is creating a `client-mu-plugins/peake-gdpr` directory for any overrides or plugin specific code. May be worth doing the same here?